homeassistant/components/cast/media_player.py

```diff
@@ -38,6 +39,8 @@
     SUPPORT_VOLUME_MUTE,
     SUPPORT_VOLUME_SET,
 )
+from homeassistant.components.plex.const import PLEX_URI_SCHEME
+from homeassistant.components.plex.services import lookup_plex_media
```

Comment:
Did you forget to commit an update to this file? I cant find lookup_plex_media in homeassistant.components.plex.services

Reply:
This is a draft that's dependent on #41505. That PR includes the function.

Reply:
Apologies, didn't see this was a draft.